keyworker-specs/src/test/groovy/uk/gov/justice/digital/hmpps/keyworker/mockapis/Elite2Api.groovy

```diff
@@ -241,6 +241,7 @@ class Elite2Api extends WireMockRule {
                 delete(urlPathEqualTo("/api/users/${username}/caseload/NWEB/access-role/${roleCode}"))
                         .willReturn(
                         aResponse()
+                                .withHeader('Content-Type', 'application/json')
```

Comment:
stubs now required to return json otherwise superagent treats it as text instead